File: src/main/java/ch/simas/monitor/entity/Measurement.java
package ch.simas.monitor.entity;

import com.fasterxml.jackson.annotation.JsonIgnore;
import java.io.Serializable;
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity
public class Measurement implements Serializable {

    private final static DateTimeFormatter DTF = DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm:ss");
    
    @Id
    @GeneratedValue
    private Integer id;
    private String name;
    private String url;
    private String status;
    private Long duration;
    private LocalDateTime timestamp;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public Long getDuration() {
        return duration;
    }

    public void setDuration(Long duration) {
        this.duration = duration;
    }

    @JsonIgnore
    public LocalDateTime getTimestamp() {
        return timestamp;
    }

    public void setTimestamp(LocalDateTime timestamp) {
        this.timestamp = timestamp;
    }

    public String getFormattedTimestamp() {
        return DTF.format(timestamp);
    }

    public String getIsoTimestamp() {
        return DateTimeFormatter.ISO_DATE_TIME.format(timestamp);
    }
}

File: src/main/java/ch/simas/monitor/entity/Measurement.kt
package ch.simas.monitor.entity

import com.fasterxml.jackson.annotation.JsonIgnore
import java.io.Serializable
import java.time.LocalDateTime
import java.time.format.DateTimeFormatter
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.Id

@Entity
open class Measurement(@Id @GeneratedValue var id: Int = 0,
                       var name: String = "",
                       var url: String = "",
                       var status: String = "",
                       var duration: Long? = 0,
                       var timestamp: LocalDateTime = LocalDateTime.now()) {

    val formattedTimestamp: String
        get() = DateTimeFormatter.ofPattern("dd.MM.yyyy HH:mm:ss").format(timestamp)

    val isoTimestamp: String
        get() = DateTimeFormatter.ISO_DATE_TIME.format(timestamp)

}
